packages/@textlint/text-to-ast/src/plaintext-parser.ts

```diff
@@ -72,7 +73,7 @@ function createBRNode(lineNumber, startIndex) {
  * @param {[TxtNode]} nodes
  * @returns {TxtNode} Paragraph node
  */
-function createParagraph(nodes) {
+function createParagraph(nodes: [TxtNode]): TxtNode {
```

Comment:
```suggestion
function createParagraph(nodes: TxtNode[]): TxtNode {
```

`TxtNode[]` is Array of TxtNode.
`[TxtNode]` is an item in a array.

Reply:
I also fixed jsdoc.